examples/applications/plot_topics_extraction_with_nmf_lda.py

```diff
@@ -38,13 +39,25 @@
 n_top_words = 20
 
 
-def print_top_words(model, feature_names, n_top_words):
+def plot_top_words(model, feature_names, n_top_words):
+    fig, axes = plt.subplots(2, 5, figsize=(30, 15), sharex=True)
+    axes = axes.flatten()
     for topic_idx, topic in enumerate(model.components_):
-        message = "Topic #%d: " % topic_idx
-        message += " ".join([feature_names[i]
-                             for i in topic.argsort()[:-n_top_words - 1:-1]])
-        print(message)
+        top_features_ind = [
+                i for i in topic.argsort()[:-n_top_words - 1:-1]]
+        top_features = [feature_names[i] for i in top_features_ind]
+        values = [topic[i] for i in top_features_ind]
+        ax = axes[topic_idx]
+        ax.barh(top_features, values)
+        ax.set_title(f'Topic {topic_idx +1}', fontdict={'fontsize': 40})
+        ax.invert_yaxis()
+        ax.tick_params(axis='both', which='major', labelsize=20)
+        for i in 'top right left'.split():
+            ax.spines[i].set_visible(False)
+
     print()
+    plt.subplots_adjust(top=0.95, bottom=0.05, wspace=0.90, hspace=0.3)
```

Comment:
Cool plots !
It is hard to disentangle the three plots though. Maybe you could add a title for each plot, like 
```suggestion
    plt.subplots_adjust(top=0.90, bottom=0.05, wspace=0.90, hspace=0.3)
    plt.suptitle(title)
```
where `title` is given as a parameter, and contains 
```
"Topics in NMF model (Frobenius norm)"
"Topics in NMF model (generalized Kullback-Leibler divergence)"
"Topics in LDA model"
```

Reply:
Thanks. Great suggestions. Updated it.